spec/views/admin/posts/new.html.haml_spec.rb

```diff
@@ -5,7 +5,7 @@
     assign(:post, create(:post))
   end
 
-  it 'renders new admin_post form' do
+  pending 'renders new admin_post form' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.